bower.json

```diff
@@ -18,11 +18,12 @@
     "testem.json"
   ],
   "dependencies": {
-    "ember-data": "1.0.0-beta.17"
+    "ember-data": "1.13.5"
```

Comment:
Yep, a `~1.13.5` would be nice here since minor changes are being released frequently right now.